lib/matplotlib/axes/_axes.py

```diff
@@ -2670,8 +2670,9 @@ def pie(self, x, explode=None, labels=None, colors=None,
         Notes
         -----
         The pie chart will probably look best if the figure and axes are
-        square, or the Axes aspect is equal.
+        square, or the Axes aspect is equal, this is the default setting.
```

Comment:
Its not the default setting, its the only setting.  Suggest:

```
This method sets the aspect ratio fo the axis to ``equal``.
```

Reply:
I was trying to say that is the default setting for pie, but I think what you said is more clear, thanks